### DISCUSSION TITLE
Add break-word class

### DISCUSSION THREAD

**Discussion by maciekgrzybek** [tailwindlabs/tailwindcss#5771] [Ideas]
Wrod break utils are not covering the `break-word` property. From my experience it's commonly used. Would it make sense to add something like:
```
.break-word {
   word-break: break-word;
}
```

**Comment by adamwathan** [comment]
Hey! So the reason we don't currently included it is because it is a deprecated CSS feature:

![image](https://user-images.githubusercontent.com/4323180/137113667-0fa9b197-2d67-4e7f-846c-46df3985a8a4.png)

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#values

Can you give an example of where it's needed that can't be solved with our existing utilities?

**Reply by maciekgrzybek** [reply]
Hey @adamwathan thanks for a quick response, I actually didn't know that :) I can replace it in that case, but still, Tailwind doesn't have anything for `overflow-wrap: anywhere;`am I correct?

**Reply by jokull** [reply]
I've come up with a case where it's actually useful to have `word-break: break-word;` as well:

Check out this [Codepen](https://codepen.io/jokull/pen/XWeaazm). It seems to be related to being inside `<td>` or `display: table-cell;` elements - in that case a `word-break: break-word;` is needed to get wrapping. If I place that `table-cell` element inside a flex container it works again! Tested in Firefox and Safari.
